vllm/model_executor/models/gpt_bigcode.py

```diff
@@ -248,6 +259,7 @@ def forward(
 class GPTBigCodeForCausalLM(nn.Module, SupportsLoRA, SupportsPP):
     packed_modules_mapping = {"c_attn": ["c_attn"]}
 
+    supported_lora_modules = ["c_fc", "c_proj", "wte", "lm_head", "c_attn"]
```

Comment:
`supported_lora_modules` was removed from all model definitions in https://github.com/vllm-project/vllm/pull/13166, is it necessary to add it here?

Reply:
Thanks for pointing this out, I've removed it. This change was older than the PR that was merged.